src/test/java/remi/model/commands/CommandListTest.java

```diff
@@ -0,0 +1,29 @@
+package remi.model.commands;
+
+import org.junit.jupiter.api.Test;
+import remi.model.TaskList;
+import remi.model.Ui;
+import remi.utils.RemiError;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class CommandListTest {
+    @Test
+    public void addTodo_NoError() {
```

Comment:
I think the test case naming should follow the format `nameOfMethodTested_inputDescription_expectedOutput`. You are missing the method name here.